docs/index.md

```diff
@@ -1,6 +1,6 @@
 <center>![images/logo.svg](images/logo.svg)</center>
 
-# RedisGraph 
+# RedisGraph
```

Comment:
I appreciate your attention to trailing whitespace!

Reply:
that's my text editor, should I leave it out? :-)